editor/export/editor_export_plugin.cpp

```diff
@@ -285,6 +285,17 @@ String EditorExportPlugin::_get_export_option_warning(const Ref<EditorExportPlat
 	return ret;
 }
 
+void EditorExportPlugin::_get_export_option_overrides(const Ref<EditorExportPlatform> &p_platform, HashMap<StringName, Variant> *r_value_overrides) const {
+	Dictionary ret;
```

Comment:
> In the current implementation, export plugin will override the overrides added by earlier export plugins, but we don't have any control over the order that they run in.

If we return the `Dictionary` value here directly, we can use it to compare the keys returned by the export plugins.
At the very least we can print a verbose warning when we detect that they are duplicated / conflicting keys.

Reply:
In my latest push I've switched it to return a `Dictionary` directly, and to output a warning if a later export plugin overrides the same option with a different value.